src/lib.rs

```diff
@@ -495,6 +537,37 @@ mod tests {
         });
     }
 
+    #[test]
+    fn basic_env_file_with_relative_dir() {
+        figment::Jail::expect_with(|jail| {
+            jail.set_env("FIGMENT_TEST_FOO_FILE", "secret");
+            jail.create_directory("subdir")?;
+            jail.create_file("subdir/secret", "bar")?;
+
+            let config = figment::Figment::new()
+                .merge(FileAdapter::wrap(Env::prefixed("FIGMENT_TEST_")).relative_dir("subdir"))
+                .extract::<Config>()?;
+
+            assert_eq!(config.foo, "bar");
+            Ok(())
+        });
+    }
+
+    #[test]
+    fn basic_env_file_with_absolute_dir() {
+        figment::Jail::expect_with(|jail| {
+            jail.set_env("FIGMENT_TEST_FOO_FILE", "secret");
+            let abs_dir = jail.create_directory("subdir")?;
+            jail.create_file("subdir/secret", "bar")?;
+
```

Comment:
Can you `assert!(abs_dir.is_absolute());` as documentation?

Reply:
done